transcom/mymove
# B 21000 int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21000)

## Summary
We want to give the customer the following capabilities:
Allow the customer to edit the counseling office.
Allow the customer to review the counseling office.


### How to test

1. Login as a Customer
2.Create a move with Origin duty location as Scott AFB
3. Select a Counseling office for the move and submit as normal

1. Select to view current move 
2. Verify the counseling office is displayed in the orders summary
3. select edit on the orders summary
4. change the counseling office and submit
5. verify the counseling office is displayed in the orders summary
6. selected edit on the order summary
7. change the origin duty location to NS Mayport
8. verify the counseling office is blank in the orders summary


## 508 Screenshots
[B-21000-508Screenshots.pdf](https://github.com/user-attachments/files/17263226/B-21000-508Screenshots.pdf)
